### ISSUE TITLE
Unit test failing due to recent upstream change

### ISSUE THREAD

**Issue by pauldamiani** [gjb2048/moodle-format_topcoll#49]
Hi gjb2048,

A patch that was recently accepted into core has caused unit test failures in courseformatrenderer_test.php on tests test_toggle_all and test_display_instructions due to an additional aria-hidden="true" tag being added. Moodle 3.5 and up are affected.

Tracker: https://tracker.moodle.org/browse/MDL-62680
Diff: https://github.com/damyon/moodle/compare/4c6ae316d8...MDL-62680-35

Regards,
Paul

**Comment by gjb2048** [comment]
Thanks for the information Paul, the issue has already been raised here: https://github.com/gjb2048/moodle-format_topcoll/pull/48
